otter/controller.py

```diff
@@ -306,17 +316,25 @@ def modify_state(group_info):
 @defer.inlineCallbacks
 def modify_and_trigger(dispatcher, group, logargs, modifier, *args, **kwargs):
     """
-    Modify group state and trigger convergence after that
+    Modify group state and trigger convergence after that. Checks if group is
```

Comment:
In other words, just saying "checks" is generally bad - the caller doesn't care whether the implementation "checks" or not, they care about what happens when the check fails or succeeds.